docs/content/querying/timeboundaryquery.md

```diff
@@ -9,6 +9,7 @@ Time boundary queries return the earliest and latest data points of a data set.
     "queryType" : "timeBoundary",
     "dataSource": "sample_datasource",
     "bound"     : < "maxTime" | "minTime" > # optional, defaults to returning both timestamps if not set 
+    "filter"    : { "type": "and", "fields": [<filter>, <filter>, ...] } # optional
```

Comment:
I couldn't find an example of just using `"filter": <filter>` anywhere else in the docs, for example in TimeSeries an actual filter is given. I can change it, just wanted to consistent.